hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -296,7 +297,7 @@ public class DatanodeConfiguration extends ReconfigurableConfig {
           " Either of min.free.space or min.free.space.percent should be configured, when both are set then" +
           " min.free.space will be used."
   )
-  private float minFreeSpaceRatio = MIN_FREE_SPACE_UNSET;
+  private float minFreeSpaceRatio = HDDS_DATANODE_VOLUME_MIN_FREE_SPACE_PERCENT_DEFAULT;
```

Comment:
We shall update update minFreeSpace default value too. 
Also please update the description for "hdds.datanode.volume.min.free.space.percent" and "hdds.datanode.volume.min.free.space" that both both properties will be considered in final free space calculation.

Reply:
updated